app/controls/drag-drop.js

```diff
@@ -1,10 +1,7 @@
-import API from 'ipfs-api'
 import notifier from 'node-notifier'
 import {join} from 'path'
 import {getIPFS, logger} from './../init'
```

Comment:
this works in Node.js due to transpilation, correct? Do we want to use two module systems in the same file? I'ld rather prefer to keep using CJS on Node.js side and ES6 for browser/ui side.